onedal/datatypes/tests/test_data.py

```diff
@@ -144,6 +145,19 @@ def test_input_format_f_contiguous_pandas(queue, dtype):
     _test_input_format_f_contiguous_pandas(queue, dtype)
 
 
+def _test_conversion_to_table(dtype):
+    x = np.random.randint(0, 10, (15, 3), dtype=dtype)
+    x_table = to_table(x)
+    x2 = from_table(x_table)
+    assert x.dtype == x2.dtype
+    assert np.array_equal(x, x2)
+
+
+@pytest.mark.parametrize("dtype", [np.int32, np.int64])
+def test_conversion_to_table(dtype):
+    _test_conversion_to_table(dtype)
```

Comment:
please define seed make this for both gpu and cpu and extend types for testing

Reply:
We don't use information about queue to use methods to_table and from_table so these tests are suitable for both cpu and gpu